docs/hub/security.md

```diff
@@ -165,3 +165,18 @@ git commit -S -m "My first signed commit"
 ```
 
 Once pushed on the Hub, you should see the commit with a "Verified" badge.
+
+## Malware Scanning
+
+We run every file of your repositories through a [malware scanner](https://www.clamav.net/).
+
+Scanning is triggered at each commit or when you visit a repository page.
+
+Here is an example view of an infected file :
+
+![/docs/assets/hub/eicar-hub-tree-view.png](/docs/assets/hub/eicar-hub-tree-view.png)
+
+![/docs/assets/hub/eicar-hub-file-view.png](/docs/assets/hub/eicar-hub-file-view.png)
+
```

Comment:
maybe even link to that dummy sample repo here?

Reply:
Yes why not :)